x-pack/plugins/lens/public/shared_components/coloring/utils.ts

```diff
@@ -293,6 +347,52 @@ export function getColorStops(
   return freshColorStops;
 }
 
+/**
+ * Some name conventions here:
+ * * `stops` => final steps used to table coloring. It is a rightShift of the colorStops
+ * * `colorStops` => used for correct work other part of application based on `stops`.  Used to compute range min.
+ * * `colorRanges` => user's color ranges inputs.  Used to compute colorStops. The main diff here we have completely range for each color.
+ *
+ * Both table coloring logic and EuiPaletteDisplay format implementation works differently than our current `colorStops`,
+ * by having the stop values at the end of each color segment rather than at the beginning: `stops` values are computed by a rightShift of `colorStops`.
+ * EuiPaletteDisplay has an additional requirement as it is always mapped against a domain [0, N]: from `stops` the `displayStops` are computed with
+ * some continuity enrichment and a remap against a [0, 100] domain to make the palette component work ok.
+ *
+ * These naming conventions would be useful to track the code flow in this feature as multiple transformations are happening
+ * for a single change.
+ */
```

Comment:
do all these conventions still apply?

Reply:
I'm not a big fan of adding comments about name conventions in method comments. If we really need it let's add some comment for Types. 
 
The following part was removed:

> * Some name conventions here:
>  * * `stops` => final steps used to table coloring. It is a rightShift of the colorStops
>  * * `colorStops` => used for correct work other part of application based on `stops`.  Used to compute range min.
>  * * `colorRanges` => user's color ranges inputs.  Used to compute colorStops. The main diff here we have completely range for each color.